ci/Dockerfile.rpm

```diff
@@ -0,0 +1,7 @@
+# syntax=docker/dockerfile:1
+
+# For building and testing rpm files in GitHub Action pipeline.
+
+FROM registry.suse.com/bci/bci-base
+
+RUN zypper install -y libmount-devel gcc make rpm-build git sudo
```

Comment:
Maybe we can just run this inside the action? It takes 15 seconds, and then we don't have to worry about `bcumming/bci-base-rpmbuilder:testing` having an outdated base image layer